lua/telescope/_extensions/file_browser/config.lua

```diff
@@ -62,6 +62,39 @@ _TelescopeFileBrowserConfig = {
 
 config.values = _TelescopeFileBrowserConfig
 
+local hijack_netrw = function()
+  vim.g.loaded_netrw = 1
+  vim.g.loaded_netrwPlugin = 1
+
+  vim.api.nvim_create_augroup("FileExplorer", { clear = true })
+  vim.api.nvim_create_autocmd("BufEnter", {
+    group = "FileExplorer",
+    pattern = "*",
+    callback = function()
+      vim.schedule(function()
+        local bufname = vim.api.nvim_buf_get_name(0)
+        if vim.fn.isdirectory(bufname) == 0 then
+          config.values.netrw_bufname = vim.fn.expand "#:p:h"
+          return
+        end
+
+        -- prevents reopening of file-browser if exiting without selecting a file
+        if config.values.netrw_bufname == bufname then
```

Comment:
Can't this be just a local variable outside the `hijack` function? It's not part of the config as such and ideally shouldn't be.